introduction.md

```diff
@@ -0,0 +1 @@
+Hello, my name is Heather. I'm here to chew bubblegum and master Github. And I'm all out of bubblegum.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.